open-company/open-company-web
# Ms Edge fixes

Bug: Microsoft Edge has a [known bug](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15534521/) on scrollTo function, i replaced all those calls with `(set! (.. js/document -scrollingElement -scrollTop) scroll-to-y)` only for this browser.

To test:
- load this branch with Chrome
- scroll down the stream
- click on a post
- [ ] do you see the top of the expanded post without having to scroll back up? Good
- no click the back to AP (or section name) button
- [ ] are you at the same scroll point where you left? Good
- repeat on Edge to make sure it works
